views/edit-program.ejs

```diff
@@ -61,12 +61,23 @@
 					<% } %>
 				</select>
 			</div>
+			<div class="form-group col-md-6">
+				<label for="student_list">Students Involved</label>
+				<select class="form-control" name="student_list" id="student_list" multiple>
+					<% for (let i = 0; i < students.length; i++) { %>
+						<option><%= students[i].first_name%> - <%= students[i].id %> </option>
```

Comment:
When editing, is it possible to make the currently enrolled students selected in the box?